web3_proxy/src/frontend/admin.rs

```diff
@@ -137,29 +136,14 @@ pub async fn admin_increase_balance(
     balance::Entity::insert(balance_entry)
         .on_conflict(
             OnConflict::new()
-                .values([
-                    // (
-                    //     balance::Column::Id,
-                    //     Expr::col(balance::Column::Id).add(self.frontend_requests),
-                    // ),
-                    (
-                        balance::Column::AvailableBalance,
-                        Expr::col(balance::Column::AvailableBalance).add(amount),
-                    ),
-                    // (
-                    //     balance::Column::Used,
-                    //     Expr::col(balance::Column::UsedBalance).add(self.backend_retries),
-                    // ),
-                    // (
-                    //     balance::Column::UserId,
-                    //     Expr::col(balance::Column::UserId).add(self.no_servers),
-                    // ),
-                ])
+                .values([(
+                    balance::Column::AvailableBalance,
+                    Expr::col(balance::Column::AvailableBalance).add(amount),
+                )])
```

Comment:
https://github.com/llamanodes/web3-proxy/pull/84#discussion_r1210950758

Reply:
The origin of this code snippet is from the other PR, so I will resolve this comment here